4MI0800264/Problem 2.cpp

```diff
@@ -0,0 +1,103 @@
+#include <iostream>
+using namespace std;
+
+struct OrderedPair
+{
+	int first, second;
+	/*void read();
+	void print();*/
+};
+//void OrderedPair::read()
+//{
+//	cout << "Enter natural numbers for ordered pair >";
+//	cin >> first >> second;
+//}
+//void OrderedPair::print() 
+//{
+//	cout << first << " " << second << endl;
+//}
+
+void readOrderedPair(OrderedPair& pair)
+{
+	cout << "Enter natural numbers for ordered pair >";
+	cin >> pair.first >> pair.second;
+}
+
+void printOrderedPair(const OrderedPair& pair)
+{
+	cout << pair.first << " " << pair.second << endl;
+}
+
+int compareOrderedPairs(const OrderedPair& pair1, const OrderedPair& pair2)
+{
+	if (pair1.first > pair2.first || (pair1.first == pair2.first && pair1.second > pair2.second))
+		return 1;
+	else if (pair1.first == pair2.first && pair1.second == pair2.second)
+		return 0;
+	else
+		return -1;
+}
+
+OrderedPair findMaxOrderedPair(const OrderedPair array[100])
+{
+	OrderedPair max = array[0];
+
+	for (int i = 0; i < 9; i++)
```

Comment:
You mean 99, right ?

Reply:
Yes, I meant to write 99, but since the array was not filled with ordered pairs, and the automatically filled ones are with random integers, the function did not work as intended. I must have forgotten to change it back to 99.